src/fsharp/TypeChecker.fs

```diff
@@ -1405,31 +1405,32 @@ let ComputeAccessAndCompPath env declKindOpt m vis overrideVis actualParent =
     let cpath = if accessModPermitted then Some env.eCompPath else None
     vis, cpath 
 
-let CheckForAbnormalOperatorNames cenv (idRange:range) opName isMember =    
+let CheckForAbnormalOperatorNames cenv (idRange:range) coreDisplayName memberInfoOpt =
     if (idRange.EndColumn - idRange.StartColumn <= 5) && 
         not cenv.g.compilingFslib 
     then
+        let opName = DecompileOpName coreDisplayName
         match opName with 
         | PrettyNaming.Relational ->
-            if isMember then
-                warning(StandardOperatorRedefinitionWarning(FSComp.SR.tcInvalidMethodNameForRelationalOperator(opName, (CompileOpName opName)), idRange))
+            if Option.isSome memberInfoOpt then
```

Comment:
Can we bring this up? ex: `let isMember = Option.isSome memberInfoOpt` and use `isMember`

Reply:
done.

Reply:
I would prefer ``memberInfoOpt.IsSome`` to `Option.isSome memberInfoOpt`.  Use properties if they are there